templates/layout.html

```diff
@@ -5,7 +5,7 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>Ubuntu Insights {% block title %}{% endblock %}</title>
```

Comment:
Could I suggest:

``` jinja2
    {% if post.title %}
      <title>{{ post.title.rendered }} | Ubuntu Insights</title>
    {% else %}
      <title>Ubuntu Insights</title>
    {% endif %}
```

It seems clearer to me what's going on. Thoughts?

Reply:
+1, I'll do another PR to fix it